tasks.json

```diff
@@ -0,0 +1 @@
rmission", "pk": 31, "fields": {"name": "Can delete task", "content_type": 8, "codename": "delete_task"}}, {"model": "auth.permission", "pk": 32, "fields": {"name": "Can view task", "content_type": 8, "codename": "view_task"}}, {"model": "auth.user", "pk": 1, "fields": {"password": "pbkdf2_sha256$720000$WpxeTqHuS724LLMC5wUdSZ$C2C0ydNKLBCK2K+JkzEyFgW2bgNUvqmPjbx5d8biAZ4=", "last_login": "2024-04-10T11:50:53.941Z", "is_superuser": true, "username": "admin@example.com", "first_name": "", "last_name": "", "email": "admin@example.com", "is_staff": true, "is_active": true, "date_joined": "2024-04-10T11:50:18.522Z", "groups": [], "user_permissions": []}}, {"model": "contenttypes.contenttype", "pk": 1, "fields": {"app_label": "admin", "model": "logentry"}}, {"model": "contenttypes.contenttype", "pk": 2, "fields": {"app_label": "auth", "model": "permission"}}, {"model": "contenttypes.contenttype", "pk": 3, "fields": {"app_label": "auth", "model": "group"}}, {"model": "contenttypes.contenttype", "pk": 4, "fields": {"app_label": "auth", "model": "user"}}, {"model": "contenttypes.contenttype", "pk": 5, "fields": {"app_label": "contenttypes", "model": "contenttype"}}, {"model": "contenttypes.contenttype", "pk": 6, "fields": {"app_label": "sessions", "model": "session"}}, {"model": "contenttypes.contenttype", "pk": 7, "fields": {"app_label": "task", "model": "tag"}}, {"model": "contenttypes.contenttype", "pk": 8, "fields": {"app_label": "task", "model": "task"}}, {"model": "sessions.session", "pk": "te3jmz86glxaipmgwtntoy504hl5rjis", "fields": {"session_data": ".eJxVjEEOgkAMRe8yazMBC52pS_ecYdJOi6AGEgZWxrsrCQvd_vfef7nE2zqkrdiSRnUXV7vT7yacHzbtQO883Waf52ldRvG74g9afDerPa-H-3cwcBm-dQwxZ2OiOjQhRMwIdEbWFtREQmO9SG4hVkgxQg8AWhGwItZoRL17fwDdEzek:1ruWTd:MzDX5URQ54iilQhzJ6lTEP-L8zY6DwhcfW5OU2rNiJw", "expire_date": "2024-04-24T11:50:53.951Z"}}, {"model": "task.tag", "pk": 1, "fields": {"name": "cat"}}, {"model": "task.tag", "pk": 2, "fields": {"name": "pet"}}, {"model": "task.tag", "pk": 6, "fields": {"name": "oduvanchik2"}}, {"model": "task.tag", "pk": 7, "fields": {"name": "platon"}}, {"model": "task.task", "pk": 1, "fields": {"content": "to pat a cat", "created": "2024-04-10T12:41:04.183Z", "deadline": "2024-04-30T12:40:46Z", "task_done": true, "tags": [1, 2]}}, {"model": "task.task", "pk": 2, "fields": {"content": "read the book", "created": "2024-04-10T12:41:37.192Z", "deadline": "2024-04-29T12:41:24Z", "task_done": true, "tags": []}}, {"model": "task.task", "pk": 3, "fields": {"content": "gladit psa2", "created": "2024-04-10T16:39:29.102Z", "deadline": "2024-04-21T16:39:27Z", "task_done": true, "tags": [2]}}, {"model": "task.task", "pk": 7, "fields": {"content": "погладить котика быстро", "created": "2024-04-11T14:40:14.588Z", "deadline": "2024-04-28T14:40:00Z", "task_done": false, "tags": []}}, {"model": "task.task", "pk": 9, "fields": {"content": "perchik", "created": "2024-04-11T15:52:26.172Z", "deadline": "2024-04-27T15:52:00Z", "task_done": false, "tags": [6]}}]
```

Comment:
Remove this file from PR

Reply:
this is database dump)

Reply:
There is no need to include this dump in PR